### PR TITLE
more Pandas backend options

### PR DESCRIPTION
**Pull request by nicolaskruchten**
- [x] I have added tests (if submitting a new feature or correcting a bug) or
  modified existing tests.
- [x] For a new feature, I have added documentation examples in an existing or
  new tutorial notebook (please see the doc checklist as well).
- [x] I have added a CHANGELOG entry if fixing/changing/adding anything substantial.

### CODE REVIEW

File: packages/python/plotly/plotly/__init__.py [hunk 1]
```diff
@@ -99,9 +112,27 @@ def plot(data_frame, kind, **kwargs):
...
+    if kind in ["hist", "histogram"]:
         new_kwargs = {k: kwargs[k] for k in kwargs if k not in ["by", "bins"]}
         return histogram(data_frame, **new_kwargs)
+    if kind == "violin":
+        return violin(data_frame, **kwargs)
+    if kind == "strip":
+        return strip(data_frame, **kwargs)
+    if kind == "funnel":
+        return funnel(data_frame, **kwargs)
+    if kind == "density_contour":
```
Comment (emmanuelle) [inline packages/python/plotly/plotly/__init__.py:124]: I don't really understand how `df.plot(kind='density_contour')` is supposed to work. Does it suppose it's a wide-mode or long-mode dataframe?

Comment (nicolaskruchten) [reply packages/python/plotly/plotly/__init__.py:124]: it supports wide mode, yes. it works just like scatter basically, which works just like line :)

Comment (emmanuelle) [reply packages/python/plotly/plotly/__init__.py:124]: ok

**Comment by emmanuelle** [comment]
:woman_dancing:

**Comment by emmanuelle** [comment]
(once changelog conflicts have been resolved).

**Pull request merged**